apluslms_roman/builder.py

```diff
@@ -60,11 +88,13 @@ def __init__(self, backend_class=None, settings=None):
 
         name = getattr(backend_class, 'name', None) or backend_class.__name__.lower()
         env_prefix = name.upper() + '_'
-        env = {k: v for k, v in environ.items() if k.startswith(env_prefix)}
+        env = {k:  get_pair_form_env(k, v) for k, v in environ.items() if k.startswith(env_prefix)}
```

Comment:
this and and few lines below, could be rewritten to use `load_from_environ` defined in different comment. Thus, create a dictionary, which can be then passed as it to backend class. Environment dictionary can then be converted a list of environment variables if really needed.